src/utils/fetch-youtube-data.js

```diff
@@ -1,27 +1,12 @@
-import dlv from 'dlv';
 import { requestYoutubePlaylist } from './devhub-api-stitch';
+import { simplifyYoutubeResponse } from './simplify-youtube-response';
 // Fetches data from youtube api
 
-const simplifyResponse = responseData => {
```

Comment:
These mappings needed to be pulled out of these modules entirely, since they would import from a module that uses `mongodb-stitch-browser-sdk`.

Reply:
They can (and maybe should) move into Stitch as well, seems to be more server-side logic. Thoughts?

Reply:
It would keep our end use of the API data consistent, but we'd just need to make sure it's noted that if the API from Twitch changes we need to update at the stitch level.

Reply:
Yeah I think I will tackle that separately